### PR TITLE
Create a bio for ametla

### LINKED ISSUE #48: Add bio for ametla
### Things to include

**everything I like to do**

- [ ] Travel
- [ ] Food

@cchay is there anything you think I should add?

### PR DESCRIPTION
**Pull request by ametla**
Resolves #48 

Adding a bio for ametla

@jpallepo what else do you think I should add?

Looks good.

### CODE REVIEW

File: bios/ametla-bio.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+## ametla
+
+- Lives in Fleming Island
```
Comment (aaiken1) [inline bios/ametla-bio.md:3]: Is that Florida?

Comment (ametla) [inline bios/ametla-bio.md:3]: Yes

Comment (aaiken1) [inline bios/ametla-bio.md:3]: Looks good

**Comment by ametla** [comment]
Where are you from?

**Pull request merged**